src/codegen.cpp

```diff
@@ -5753,6 +5930,20 @@ static void init_julia_llvm_env(Module *m)
                          "jl_gc_diff_total_bytes", m);
     add_named_global(diff_gc_total_bytes_func, (void*)*jl_gc_diff_total_bytes);
 
+#ifndef LLVM36
+    Type *powf_type[2] = { T_float32, T_float32 };
+    jlpowf_func = Function::Create(FunctionType::get(T_float32, powf_type, false),
+                                   Function::ExternalLinkage,
+                                   "powf", m);
+    add_named_global(jlpowf_func, (void*)&powf);
+
+    Type *pow_type[2] = { T_float64, T_float64 };
+    jlpow_func = Function::Create(FunctionType::get(T_float64, pow_type, false),
+                                  Function::ExternalLinkage,
+                                  "pow", m);
+    add_named_global(jlpow_func, (void*)&pow);
```

Comment:
that'll manage to call the "untrusted" system libm, rather than the statically linked openlibm

@yuyichao from SO, it sounds to me like that's how it was supposed to be implemented originally, but some more recent C++ standard has changed that to acknowledge that some compiler implementations just reused the C header + overloads.